lua/lazyvim/plugins/ui.lua

```diff
@@ -198,7 +198,12 @@ return {
       }
 
       -- do not add trouble symbols if aerial is enabled
-      if vim.g.trouble_lualine and LazyVim.has("trouble.nvim") then
+      -- And allow it to be overriden for some buffer types (see autocmds)
+      if
+        vim.g.trouble_luline_enabed ~= false
```

Comment:
There's a typo here? Should be `vim.g.trouble_lualine_enabled`.

Reply:
Fixed!